presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -527,12 +527,13 @@ private static Table buildTableObject(
 
     private static PrincipalPrivileges buildInitialPrivilegeSet(String tableOwner)
     {
+        PrestoPrincipal grantor = new PrestoPrincipal(USER, tableOwner);
```

Comment:
It is accurate here. If you are creating a table - you are a grantor for privileges for yourself. That's what hive is doing.